src/views/network-page.njk

```diff
@@ -95,5 +95,13 @@
     </div>
   </section>
   {% include "charts/_stats_with_selectors.njk" %}
+  <section class="tile">
+    <header>
+      <h2>All the {{texts.title}} nodes</h2>
```

Comment:
Strictly speaking `the` is unnecessary here. `All diaspora* nodes` for example is better than `All the diaspora* nodes`, as an English speaker.